test/controllers/reasons/UnderpaymentReasonAmendmentControllerSpec.scala

```diff
@@ -90,16 +91,16 @@ class UnderpaymentReasonAmendmentControllerSpec extends ControllerSpecBase {
     }
 
     "should redirect the back button to Box Number Controller" in new Test {
-      controller.backLink(22) mustBe Some(controllers.reasons.routes.BoxNumberController.onLoad())
+      controller.backLink(BoxNumber.Box22) mustBe Some(controllers.reasons.routes.BoxNumberController.onLoad())
     }
 
     "should redirect the back button to Item Number Controller" in new Test {
-      controller.backLink(33) mustBe Some(controllers.reasons.routes.ItemNumberController.onLoad())
+      controller.backLink(BoxNumber.Box33) mustBe Some(controllers.reasons.routes.ItemNumberController.onLoad())
     }
 
-    "should redirect the back button to Box Number Controller when the box number is not in the list" in new Test {
-      controller.backLink(0) mustBe Some(controllers.reasons.routes.BoxNumberController.onLoad())
-    }
+//    "should redirect the back button to Box Number Controller when the box number is not in the list" in new Test {
+//      controller.backLink(0) mustBe Some(controllers.reasons.routes.BoxNumberController.onLoad())
+//    }
```

Comment:
I've commented out some tests because the cases they were testing can't happen anymore,
they can all potentially be removed